ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -71,14 +71,20 @@ def check(self, _):
             queue_manager = connection.get_queue_manager_connection(self._config, self.log)
             self.service_check(self.SERVICE_CHECK, AgentCheck.OK, self._config.tags, hostname=self._config.hostname)
         except Exception as e:
-            self.warning("cannot connect to queue manager: %s", e)
+            message = 'cannot connect to queue manager: {}'.format(e)
+            self.warning(message)
             self.service_check(
-                self.SERVICE_CHECK, AgentCheck.CRITICAL, self._config.tags, hostname=self._config.hostname
+                self.SERVICE_CHECK,
+                AgentCheck.CRITICAL,
+                self._config.tags,
+                message=message,
+                hostname=self._config.hostname,
             )
             self.service_check(
                 QueueMetricCollector.QUEUE_MANAGER_SERVICE_CHECK,
                 AgentCheck.CRITICAL,
                 self._config.tags,
+                message=message,
```

Comment:
Could we update [the tests](https://github.com/DataDog/integrations-core/blob/master/ibm_mq/tests/test_ibm_mq_unit.py#L45) to include this message?

Reply:
There's no way atm to assert a prefix